add-on/panels/js/panel.jsx

```diff
@@ -308,6 +312,10 @@ loop.panel = _.extend(loop.panel || {}, (function(_, mozL10n) {
              title={mozL10n.get("settings_menu_button_tooltip")} />
           <ul className={cx({ "dropdown-menu": true, hide: !this.state.showMenu })}>
             <SettingsDropdownEntry
+                extraCSSClass="entry-settings-username entries-divider"
+                label={"Change my username"}
```

Comment:
Please add this as a .properties file entry now, and add an XXX there if its not a final version. I'd rather have these in the .properties files than not find them later on.